### ISSUE TITLE
reindex single record: location fields

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/bibdata#1040]
child of #748 - see implementation notes there

depends on #823

This issue mirrors #823 except that one indexes records from the publishing job and this one from the bibliographic#update endpoints. Values corresponding to those indexed for #823 need to be added to the marc record pulled from the API at this endpoint so that traject can index them.

**Comment by hackartisan** [comment]
closing in favor of #1478
